linux_os/guide/system/network/network-nftables/service_nftables_enabled/rule.yml

```diff
@@ -34,9 +34,21 @@ ocil: |-
 fixtext: |-
     {{{ fixtext_service_enabled("nftables") }}}
 
-platform: system_with_kernel and package[nftables] and service_disabled[firewalld]
 
+platform: machine and package[nftables] and service_disabled[firewalld]
```

Comment:
Did you intent go back to `machine` vs `system_with_kernel`?

Reply:
That was a miss during rebase thanks for raising the flag